config/routes.rb

```diff
@@ -52,6 +52,8 @@
           get :info
         end
       end
+
+      post 'sessions/start', to: 'sessions#start', as: :session_start
```

Comment:
lets try to be more restful here:

```suggestion
      resource :session, only: :create
```